src/app/template-library/template-experience-card/template-experience-card.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Template} from '../../template-library/template-library.service';
+
+@Component({
+  selector: 'app-template-experience-card',
+  templateUrl: './template-experience-card.component.html',
+  styleUrls: ['./template-experience-card.component.scss'],
+})
+export class TemplateExperienceCardComponent implements OnInit {
```

Comment:
Let's rename the component from `TemplateExperienceCard` to `TemplateCard`.
Although behind the scenes template is exported from experience, but once exported, it has nothing to do with the experience. So let's try to keep those concepts separate. 
This card is a template card.